src/ui.tsx

```diff
@@ -773,17 +773,14 @@ class UserInterface extends Component<Props, State> {
           <BaseIconButton
             tooltip={Tools.undo}
             onClick={this.undo}
-            fill={false}
-            enabled={this.state.canUndoRedo.undo}
+            fill={this.state.buttonClicked === Tools.undo.name}
```

Comment:
I'm not sure this works? You can only undo if you've already pressed undo?

Reply:
(You can actually click it anytime, but it only _looks_ enabled once you've clicked it?